### ISSUE TITLE
`attributes_table` should be available in sidebars

### ISSUE THREAD

**Issue by seanlinsley** [activeadmin/activeadmin#3314]
I've found this to be a common pattern for show pages, to fit more information:
```ruby
sidebar 'Details', only: :show do
  attributes_table_for resource do
    row :foo
    row :bar
    row :baz
  end
end

show do
  attributes_table do
    row :one
    row :two
    row :three
  end
  active_admin_comments
end
```

For some reason `attributes_table` isn't available in the sidebar, so I have to use `attributes_table_for resource` instead. IIRC this was a sticking point when I was first learning how to use Active Admin; it'd be nice to have a consistent DSL in both places.

**Comment by scarver2** [comment]
@seanlinsley @timoschilling  Here is the pull request for this feature. It includes the tests.

https://github.com/activeadmin/activeadmin/pull/4054/files#r36230933

**Comment by timoschilling** [comment]
closed by #4054
